system-parachains/bridge-hubs/bridge-hub-kusama/tests/tests.rs

```diff
@@ -82,7 +82,7 @@ fn construct_extrinsic(
 	call: RuntimeCall,
 ) -> UncheckedExtrinsic {
 	let account_id = AccountId32::from(sender.public());
-	let extra: SignedExtra = (
+	let extra: TxExtension = (
```

Comment:
(we can also add `WeightReclaim` in this pipeline, though it is a test so it is less necessary)